conda/utils.py

```diff
@@ -292,6 +292,20 @@ def get_yaml():
     return yaml
 
 
+# Restores YAML 1.1 boolean flexibility.
+yaml_bool_ = {
+    'true': True, 'yes': True, 'on': True,
+    'false': False, 'no': False, 'off': False
+}
+def yaml_bool(s, passthrough=None):
+    if type(s) is bool:
```

Comment:
My view is that `isinstance` introduces unnecessary overhead when there is no possible need to deal with inheritance. I improved the performance of the solver code significantly by moving away from `isinstance`.

http://stackoverflow.com/questions/1549801/differences-between-isinstance-and-type-in-python